benchmarks/src/main/java/org/elasticsearch/benchmark/time/RoundingBenchmark.java

```diff
@@ -19,7 +19,6 @@
 package org.elasticsearch.benchmark.time;
```

Comment:
I wonder whether it makes sense to remove this benchmark entirely now. Is it still beneficial to know about Java time performance without a baseline?

Reply:
as it is in the history we can always revive it. removed.